src/NuGet/Microsoft.CodeAnalysis.CSharp.nuspec

```diff
@@ -4,6 +4,8 @@
     <id>Microsoft.CodeAnalysis.CSharp</id>
     <description>
       .NET Compiler Platform ("Roslyn") support for C#, Microsoft.CodeAnalysis.CSharp.dll.
+
+      More details at https://github.com/dotnet/roslyn/wiki/NuGet-packages
```

Comment:
Let's use an aka.ms link here so we can redirect for packages we've already shipped.